### DISCUSSION TITLE
Directory in the automation, entities, scripts list

### DISCUSSION THREAD

**Discussion by magidel** [home-assistant/frontend#19274] [Automations / scripts / scenes]
Hello everyone, let me introduce myself.
I'm a long-time home assistant user.
I have always used this fantastic platform and I am happy that each release is always better than the previous one!

I would like to make a suggestion, but I don't know if I'm writing in the right place.
In the "automations" or "entities" section (or in any other section, for example "scripts"...) could you create a function that groups into directories?
Let me explain better: I create automations (from YAML or UI) and I would like to group them logically in a dir so as to have a more structured view of all the automations I have created and retrieve them directly from the corresponding dir, a bit like the package concept.
If I have created some automations to manage the heat in the house (boiler, fan coil...) then I will create a directory called "thermo", for example, where I will have all the automations that concern heat management, or all the "entities"...or "scripst".
These dirs could then be exploded with the tree configuration, so the "search" line could search for the automation and open the tree to the corresponding folder.
It would be a neater list of automations. :)

I hope you appreciate the suggestion.
Thanks again for your work

**Comment by frenck** [comment]
Duplicate of: https://community.home-assistant.io/t/grouping-automations-on-frontend-for-organization/102771

**Comment by magidel** [comment]
Sorry for duplicate and thank you for your job! ;)
